Cargo.toml

```diff
@@ -18,6 +18,9 @@ codegen-units = 1
 panic = "abort"
 strip = "symbols"
 
+[profile.dev]
+incremental = false
+
```

Comment:
Could we set incremental off as an override e.g. from the environment so that we don't disable incremental outside of CI?

Reply:
Sure!

Reply:
Done